.evergreen.yml

```diff
@@ -1,19 +1,64 @@
+stepback: true
+command_type: test
+ignore:
+  - "*.md"
+  - ".github/*"
+
+#######################################
+#              Functions              #
+#######################################
 functions:
-  fake_function:
-    - command: shell.exec
-      params:
-        script: |
-          echo "I am a fake function"
+  get-project:
+    command: git.get_project
+    type: setup
+    params:
+      directory: parsley
+
+  yarn-build:
+    command: shell.exec
+    params:
+      working_dir: parsley
+      script: |
+        export PATH=/opt/nodejs/node-v16.13.0-linux-x64/bin:$PATH
+        yarn build:local
+
+  yarn-install:
+    command: shell.exec
+    params:
+      working_dir: parsley
+      script: |
+        export PATH=/opt/nodejs/node-v16.13.0-linux-x64/bin:$PATH
```

Comment:
Since we're using Node v16.13 we could add an `engines` section to `package.json` like we have [in Spruce](https://github.com/evergreen-ci/spruce/blob/4fd10c45ffed5e8551f73bef326efc6cebb3eb68/package.json#L233-L235)